benchmark/benchmain/main.go

```diff
@@ -25,11 +25,12 @@ go run benchmark/benchmain/main.go -benchtime=10s -workloads=all \
   -compression=on -maxConcurrentCalls=1 -traceMode=false \
   -reqSizeBytes=1,1048576 -respSizeBytes=1,1048576 \
   -latency=0s -kbps=0 -mtu=0 \
```

Comment:
Please also document how to run twice and compare between the two.

Reply:
Done. I am not sure how to describe run twice, but to update how to compare inside benchresult/main.go.